Composer/ScriptHandler.php

```diff
@@ -104,17 +104,6 @@ public static function installAssets(CommandEvent $event)
     public static function installRequirementsFile(CommandEvent $event)
     {
         $options = self::getOptions($event);
-        $appDir = $options['symfony-app-dir'];
-
-        if (!is_dir($appDir)) {
-            echo 'The symfony-app-dir ('.$appDir.') specified in composer.json was not found in '.getcwd().', can not install the requirements file.'.PHP_EOL;
-
-            return;
-        }
-
-        copy(__DIR__.'/../Resources/skeleton/app/SymfonyRequirements.php', $appDir.'/SymfonyRequirements.php');
```

Comment:
In case some requirements change after a composer update, `SymfonyRequirements.php` should be updated too accordingly, especially after an update to a major version of symfony.

So I don't see how we could get rid of the hook. What we could do is only handle dynamic requirements in there, e.g the PHP version.

But that would require to generate the file in the bundle so composer can link the dumped version, if you really don't want to have it in `app|var` folders.